### ISSUE TITLE
Schema and table names should be quoted independently

### ISSUE THREAD

**Issue by claudio-scandura** [akka/akka-projection#420]
### Versions used 
akka-projection-jdbc:1.0.0-RC3

Akka version: 2.6.8

### Expected Behavior
When specifying a schema in the `akka.projection.jdbc.offset-store` config section then the table should be properly looked up in the database as `"schema"."table"`

### Actual Behavior
The table is instead looked up as `"schema.table"` which, at least in PostgresSQL, means a table with name `schema.table` (case sensitive) within the default (or current) schema.

### Reproducible Test Case

The current IT tests won't catch this bug even if a schema name is specified in the used config in `akka.projection.jdbc.JdbcProjectionSpec.config`

### Possible solution
A proposed solution could be to preemptively quote the table name inclusive of the schema in the `Dialect` implementation class. instead of quoting it when building the query string. For instance, in `DefaultDialect` the table can be defined as:
```
private val table = schema.map(s => s""""$s"."$tableName"""").getOrElse(s""""$tableName"""")
```

NB: This may affect DBs other than Postgres.

**Comment by patriknw** [comment]
Thanks for reporting. We will look into this before the 1.0.0 final. We are anyway an idea for the Postgres dialect.
cc @renatocaval

**Comment by renatocaval** [comment]
Fixed in #422
